lib/fauna/query.rb

```diff
@@ -50,9 +50,24 @@ def event(resource, ts, action)
     ##
     # A let expression
     #
+    # Example: <code>Fauna.query { let(x: 2).in(add(1, var(:x))) }</code>.
```

Comment:
That code example doesn't seem to match the code here. It should show an example for both a block and a passed expression.